src/fs.h

```diff
@@ -140,6 +140,28 @@ static inline path PathFromString(const std::string& string)
     return std::filesystem::path(string);
 #endif
 }
+
+/**
+ * Create directory (and if necessary its parents), unless the leaf directory
+ * already exists or is a symlink to an existing directory.
+ * This is a temporary workaround for an issue in libstdc++ that has been fixed
+ * upstream [PR101510].
+ */
+static inline bool create_directories(const std::filesystem::path& p)
+{
+    if (fs::is_symlink(p) && fs::is_directory(p)) {
```

Comment:
> I prefer to keep this fully qualified, just to be sure, and entirely clear what these refer to in case other functions are wrapped.

Then, maybe
```suggestion
    if (std::filesystem::is_symlink(p) && std::filesystem::is_directory(p)) {
```
?

Reply:
Well, if we override these, we'll likely want to use the overridden versions, right?